.travis-deps.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 if [ $TRAVIS_OS_NAME = "osx" ]; then
 	brew update
-	brew install qt5 ffmpeg imagemagick sdl2 libzip libpng
+	brew install qt5 ffmpeg imagemagick sdl2 libzip libpng gcc@4.8
```

Comment:
It would make sense to only install gcc on the matrix entry where it's needed, otherwise it's just wasting time.